client/src/index.js

```diff
@@ -42,78 +41,91 @@ const styles = makeStyles(theme => ({
+  
+//     const handleClick = () => {
+//       enqueueSnackbar('I love snacks.');
+//     };
+  
+//     const handleClickVariant = variant => () => {
+//       // variant could be success, error, warning, info, or default
+//       enqueueSnackbar('This is a success message!', { variant });
+//     };
+  
+//     return (
+//       <React.Fragment>
+//         <Button onClick={handleClick}>Show snackbar</Button>
+//         <Button onClick={handleClickVariant('success')}>Show success snackbar</Button>
+//       </React.Fragment>
+//     );
+// }
+
+class Login extends React.Component { 
+
+    constructor(props) {
+        super(props);
+        this.state = {
+            password: "",
+            confirmPassword: "",
+            email: "",
+            firstName: "",
+            lastName: "",
+            isLogin: true,
+            open: false,
+            setOpen: false,
+            transition: undefined,
+        }
     }
 
-    const updatePassword = e => {
-        state.password = e.target.value
+    updatePassword = e => {
+        this.state.password = e.target.value
     }
 
-    const updateEmail = e => {
-        state.email = e.target.value
+    updateEmail = e => {
+        this.state.email = e.target.value
     }
 
-    const updateConfirmPassword = e => {
-        state.confirmPassword = e.target.value
+    updateConfirmPassword = e => {
+        this.state.confirmPassword = e.target.value
     }
 
-    const updateFirstname = e => {
-        state.firstName = e.target.value
+    updateFirstname = e => {
+        this.state.firstName = e.target.value
     }
 
-    const updateLastname = e => {
-        state.lastName = e.target.value
+    updateLastname = e => {
+        this.state.lastName = e.target.value
     }
 
-    const handleClose = () => {
-        setOpen(false);
-    };
-
-    const handleLogin = () => {
+    handleLogin = () => {
+        //console.log(Popup.handleClick)
         // Case when there is invalid input
         if (document.getElementById("email").value === "" || document.getElementById("standard-password").value === "") {
             //setTransition();
             //setOpen(true);
             console.log("Please fill in all requirements!!!")
+            document.getElementById("errorMessage").style.display = "block"
+            document.getElementById("errorMessage").innerHTML = "Please fill in all requirements!!!"
             return
         }
 
         // Case when input is valid. This function attempts to log the user in
-        login(state.email, state.password).then(() => {
+        login(this.state.email, this.state.password).then(() => {
             // Handle successful login
-            console.log("Logged in successfully")
+            //console.log("Logged in successfully")
+            window.location = '/dashboard';
         })
         .catch(err => {
+            document.getElementById("errorMessage").innerHTML = "Please check your email and password!!!"
```

Comment:
Instead of `Please check your email and password!!!` can we do `err.message`?

Reply:
okay